Source/Core/Mixin.lua

```diff
@@ -58,10 +61,16 @@ function AuctionatorAHFrameMixin:OnEvent(eventName, ...)
   end
 end
 
-function AuctionatorAHFrameMixin:ToggleShoppingLists()
+AuctionatorButtonFrameMixin = {}
+
+function AuctionatorButtonFrameMixin:ToggleShoppingLists()
   if AuctionatorShoppingLists:IsVisible() then
     AuctionatorShoppingLists:Hide()
   else
     AuctionatorShoppingLists:Show()
   end
+end
+
+function AuctionatorButtonFrameMixin:AutoScan()
+  Auctionator.State.ScanFrameRef:InitiateScan()
```

Comment:
The events may be unregistered at this point (they usually clear after the scan)